environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentCreationValidator.java

```diff
@@ -55,6 +60,10 @@ private void validateNetwork(EnvironmentCreationDto request, String cloudPlatfor
         }
     }
 
+    private boolean isInvalidNetworkMask(String networkCidr) {
+        return !networkCidr.split("/")[1].equals(EXPECTED_NETWORK_MASK);
```

Comment:
I think we should check if the cidr actually contains the subnet mask and it's not just an IP address.

Reply:
Good idea. Please check now.

Reply:
there is a validator for this, take a look on that code ;)

Reply:
I`ve updated wrong validation. Check it again please.

Reply:
Can you please use the SubnetValidator?